docs/3.0rc/resources/upgrade-prefect-3.mdx

```diff
@@ -0,0 +1,233 @@
+---
+title: Upgrade to Prefect 3
+description: Learn how to upgrade from Prefect 2 to Prefect 3.
+---
+
+Prefect 3 is the most flexible, powerful, and fastest version of Prefect yet.
```

Comment:
super minor - to me this reads like it says "most flexible, most powerful, and most fastest"

Reply:
no thats good feedback. Fixed it. Thanks!